vmware-base/src/com/cloud/hypervisor/vmware/util/VmwareHelper.java

```diff
@@ -598,6 +601,19 @@ public static void setBasicVmConfig(VirtualMachineConfigSpec vmConfig, int cpuCo
         vmConfig.setGuestId(guestOsIdentifier);
     }
 
+    public static void addUSBController(VirtualMachineConfigSpec vmConfigSpec) {
```

Comment:
@sureshanaparti if other methods are like and you're simply follow those conventions, then it's fine though it becomes easier to test methods functionally if they return objects instead of modifying them. In either case, can you add a simple unit test for this. Thanks.